src/app/child-dev-project/children/model/child.ts

```diff
@@ -58,7 +58,7 @@ export class Child extends Entity {
    */
   @DatabaseField() photoFile: string;
 
-  @DatabaseField({ dataType: "load-child-photo" })
+  @DatabaseField({ dataType: "load-child-photo", defaultValue: true })
   photo: BehaviorSubject<SafeUrl>;
```

Comment:
Why is the default value for this field a boolean?

Reply:
the important point for the `load-child-photo` is that a default is set at all - it actually ignores the value of it but this ensures the transformation is trigger even if no value was stored. Do you have a suggestion for a more clear value?